docs/user-guide/security-context.md

```diff
@@ -1,4 +1,80 @@
----
----
+A security context defines the operating system security settings (uid, gid, capabilities, SELinux role, etc..) applied to a container. See [security context design](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/security_context.md) for more details.
```

Comment:
You have to lead the file with these lines:

---
---

That's just one of the rules of GitHub Pages/Jekyll, I'm afraid.